flowredux/src/commonMain/kotlin/com/freeletics/flowredux/sideeffects/OnActionStateMachine.kt

```diff
@@ -19,7 +19,7 @@ import kotlinx.coroutines.launch
 import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
 
-internal class StartStateMachineOnActionInStateSideEffectBuilder<SubStateMachineState : Any, SubStateMachineAction : Any, InputState : S, ActionThatTriggeredStartingStateMachine : A, S : Any, A : Any>(
+internal class OnActionStateMachine<SubStateMachineState : Any, SubStateMachineAction : Any, InputState : S, ActionThatTriggeredStartingStateMachine : A, S : Any, A : Any>(
```

Comment:
Just personal preference: `OnActionStartStateMachine ` (so add the `Start`). Your call, just a personal preference.

Reply:
[d2c9d88](https://github.com/freeletics/FlowRedux/pull/505/commits/d2c9d88aeafd019b88f8859559d1b73a052c849a)